tools/benchmark/client/cli/tx.go

```
@@ -110,6 +110,7 @@ func NewLoadTestCmd(params *modulev1.GeneratorParams) *cobra.Command {
 				}
 			}()
 
+			i := 0
```

golangci-lint (1.62.2)

var `i` is unused

(unused)